### PR TITLE
[GH-1053] Fix table horizontal lines

### LINKED ISSUE #1053: Bug: horizontal lines are truncated in table view
**Summary:**
On narrow viewport when the table view has horizontal scroll the horizontal lines for table rows are truncated.

**Expected behavior:**
Horizontal lines for rows should be full width.

**Screenshots:**
TableViewHorizontalLines

**Platform:**
 - Browser and version: Chrome 92
 - OS and version: Windows 10

### PR DESCRIPTION
**Pull request by kamre**
#### Summary
Set width to fit content for table header and row container.

#### Ticket Link
Fixes #1053

### CODE REVIEW

File: webapp/src/components/table/table.scss [hunk 1]
```diff
@@ -124,7 +124,6 @@
 
         .Editable {
             padding: 0 5px;
-            display: relative;
```
Comment (kamre) [inline webapp/src/components/table/table.scss:127]: This was just a wrong value for css display property.

**Review by sbishel — APPROVED** [review]
LGTM - Thanks for your contribution. The footer row still has issues, but this fixes the main display and looks much better.

**Comment by kamre** [comment]
@sbishel good catch! I have missed the footer row.

For the footer row width to match the width of header and row container I had to introduce intermediate `div` wrapper. Please review the changes.

File: webapp/src/components/gallery/gallery.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ const Gallery = (props: Props): JSX.Element => {
     const visibleTitle = activeView.fields.visiblePropertyIds.includes(Constants.titleColumnId)
 
     return (
-        <div className='octo-table-body Gallery'>
+        <div className='Gallery'>
```
Comment (kamre) [inline webapp/src/components/gallery/gallery.tsx:56]: `octo-table-body` class was only used for the table so it is not needed here.

**Review by hahmadia — APPROVED** [review]

**Pull request merged**